src/frontend/components/Button.spec.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { matches } from './test-utils';
+import Button from './Button';
+
+describe('Button', () => {
+  it('matches snapshot', () => {
+    matches(<Button>Sign in</Button>);
+  });
+});
```

Comment:
This button component is not suppose to be in this PR. It has already been captured in the the Button component PR